module/VuFindTheme/src/VuFindTheme/View/Helper/HeadLink.php

```diff
@@ -168,7 +168,7 @@ public function forcePrependStylesheet(
     protected function isExcludedFromConcat($item)
     {
         return !isset($item->rel) || $item->rel != 'stylesheet'
-            || !$this->isRelativePath($item->href);
+            || (isset($item->href) && !$this->isRelativePath($item->href));
```

Comment:
Would this make more sense?

```suggestion
            || !isset($item->href) || !$this->isRelativePath($item->href);
```

If there is no href, there is nothing to concatenate, so we might as well exclude it.

(I'm not sure what situation required this fix, so maybe I'm misjudging the solution... let me know if you disagree; either way, we should backport this fix to the release-10.1 branch, and we will then need to resolve conflicts with #4213, which completely rewrites all of the asset pipeline code).